src/main/app/Resources/modules/layout/sections/home/store/selectors.js

```diff
@@ -25,8 +25,14 @@ const homeData = createSelector(
   (home) => home.data
 )
 
+const areTopSubTabsVisible = createSelector(
+  [home],
+  (home) => home.are_top_sub_tabs_visible
+)
+
```

Comment:
To remove. This will make all `home` tools dependent to the platform home context.